### PR TITLE
Fix arguments to use camelcase pattern

### PR DESCRIPTION
**Pull request by carolinesalib**
A recent change on graphql-ruby gem:
Release: https://github.com/rmosolgo/graphql-ruby/releases/tag/v1.9.3
Commit: https://github.com/rmosolgo/graphql-ruby/commit/8ac51a6dcfa7c15509e88d9f0f431c6e1c500578

### CODE REVIEW

**Comment by jonatasbaldin** [comment]
Thanks for the PR!

Hey @RStankov, could you take a look?

File: content/backend/graphql-ruby/7-filtering.md [hunk 1]
```diff
@@ -71,8 +71,8 @@ class Resolvers::LinksSearch
 
   def normalize_filters(value, branches = [])
     scope = Link.all
-    scope = scope.like(:description, value['description_contains']) if value['description_contains']
-    scope = scope.like(:url, value['url_contains']) if value['url_contains']
+    scope = scope.like(:description, value['descriptionContains']) if value['descriptionContains']
```
Comment (RStankov) [inline content/backend/graphql-ruby/7-filtering.md]:
```suggestion
    scope = scope.like(:description, value[:description_contains]) if value[:description_contains]
```

File: content/backend/graphql-ruby/7-filtering.md [hunk 2]
```diff
@@ -71,8 +71,8 @@ class Resolvers::LinksSearch
...
+    scope = scope.like(:url, value['urlContains']) if value['urlContains']
```
Comment (RStankov) [inline content/backend/graphql-ruby/7-filtering.md]:
```suggestion
    scope = scope.like(:url, value[:url_contains]) if value[:url_contains]
```

**Comment by carolinesalib** [comment]
Thanks @RStankov

**Comment by jonatasbaldin** [comment]
Noice! Will merge it :)

**Pull request merged**